go.mod

```diff
@@ -19,11 +19,11 @@ require (
 	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/go-errors/errors v1.0.1
 	github.com/go-openapi/strfmt v0.19.5 // indirect
-	github.com/golang/protobuf v1.4.3
+	github.com/golang/protobuf v1.5.2
```

Comment:
Also need to update pinned dependencies CI check.

Reply:
Yes of course, done.